### PR TITLE
Adding profile type 

### PR DESCRIPTION
**Pull request by paigerube14**
Passing job: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/kube-burner-ocp/102/

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -331,18 +340,6 @@ pipeline {
...
-                branches: [[name: 'main' ]],
-                userRemoteConfigs: [[url: "https://github.com/openshift-qe/ocp-qe-perfscale-ci" ]],
-                extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'helpful_scripts']]
-            ])
-            copyArtifacts(
-                fingerprintArtifacts: true, 
-                projectName: JOB_NAME,
-                selector: specific(JENKINS_JOB_NUMBER),
-                target: 'workload-artifacts'
-            )
```
[the remark this answers is not in the document]
Comment (paigerube14) [reply Jenkinsfile:345]: yes, taking out sandman items

File: Jenkinsfile [hunk 2]
```diff
@@ -103,6 +103,11 @@ pipeline {
           Read <a href=https://github.com/openshift-qe/ocp-qe-perfscale-ci/tree/kube-burner/README.md>here</a> for details about each variable
           '''
       )
+      string(
+          name: "PROFILE_TYPE",
+          defaultValue: "both",
+          description: "Select the type of metric collection you want, values are 'both', 'reporting', and 'regular'"
+      )
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply Jenkinsfile]: first choice is default, IIRC if you don't set it default will be picked - to set it you can just pass as a string val

[the remark this answers is not in the document]
Comment (paigerube14) [reply Jenkinsfile]: gotcha, then should be all set: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/paige-e2e-multibranch/job/kube-burner-ocp/107/

**Review by nathan-weinberg — APPROVED** [review]
LGTM, one last question - I thought `regular` mode was inclusive of `reporting` mode - but `both` was suggest there are unique attributes to both?

**Comment by paigerube14** [comment]
@nathan-weinberg regular mode just gets the metrics that is defined from the metrics profile you set, reporting uses the reporting file [here](https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/config/metrics-report.yml) (which does more instant metrics of some similar metric calls so less data) and then both gets both sets of metrics. Not sure if I answered your question though..

**Pull request merged**